src/node/history.h

```diff
@@ -369,6 +382,14 @@ namespace ccf
       nodes(nodes_)
     {}
 
+    bool update_r_tree()
```

Comment:
This does not seem to update the replicated tree but only returns `true` if the consensus is PBFT. You could rename it to something like `is_replicated_tree_enabled()` or something like that.